### ISSUE TITLE
Getting NAN after few iterations

### ISSUE THREAD

**Issue by dhirajsuvarna** [charlesq34/pointnet#239]
Hi,

I am using PointNet on Pytorch using this [github](https://github.com/fxia22/pointnet.pytorch) to train my custom dataset.

However, after training around 10-20 batches of examples, the loss goes into NAN.

I highly doubt that it is a exploding gradient issue. (still trying to diagnose the problem)

Posting this problem here in expectation of any help or direction that you may provide.

PS: I have also posted this in the github of fxia11/pointnet.pytorch.

**Comment by dhirajsuvarna** [comment]
There were NAN's in the input to the network.
These NaN's were generated after the Zero Mean and Normalization step for one model only.
For time being I have removed that particular model from the dataset and was able to train.

Thank You.
